packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -0,0 +1,771 @@
+/// <reference types="Cypress" />
+
+import React from "react";
+import {
+  IcBadge,
+  IcButton,
+  IcCard,
+  IcStatusTag,
+  IcTypography,
+} from "../../components";
+import {
+  BE_VISIBLE,
+  HAVE_CLASS,
+  HAVE_FOCUS,
+  HAVE_LENGTH,
+  NOT_HAVE_FOCUS,
+} from "../utils/constants";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { mount } from "cypress/react";
+import { SlottedSVG } from "../../react-component-lib/slottedSVG";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
+
+describe("IcCard", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("renders Card", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <a href="/">
+          <IcCard heading="Card" message="This is a static card"></IcCard>
+        </a>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("a").should(HAVE_CLASS, "ic-card-wrapper-link");
+    cy.get("a").focus().should(HAVE_FOCUS);
+    cy.checkA11yWithWait();
+  });
+
+  it("should click on card", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <IcCard 
+  heading="Americano order" 
+  subheading="Name: Michael" 
+  message="Extras: Double espresso shot and oat milk." 
+  clickable
+  onClick={(ev) => console.log(ev)}>
+ <SlottedSVG
+    slot="icon"
+    xmlns="http://www.w3.org/2000/svg"
+    height="24px"
+    viewBox="0 0 24 24"
+    width="24px"
+    fill="#000000"
+  >
+   <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z"/>
+  </SlottedSVG>
+</IcCard>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("ic-card").should(BE_VISIBLE).click().should(HAVE_FOCUS);
+    cy.checkA11yWithWait();
+  });
+
+  it("should get focussed class applied when parent link focussed", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+      <a href="/">
+        <IcCard heading="Card" message="This is a static card"></IcCard>
+      </a>
+      </div>
+    );
+    cy.checkHydrated("ic-card");
+    cy.get("ic-card").shadow().find("div").should(NOT_HAVE_FOCUS);
+    cy.get("a").focus().should(HAVE_FOCUS);
+    cy.get("ic-card").shadow().find("div").should(NOT_HAVE_FOCUS);
```

Comment:
We want to be checking that the div receives the 'focussed' class, so I think the assertions after focusing the parent tag should be;
 cy.get("ic-card").shadow().find("div").should(HAVE_CLASS, 'focussed');